library/src/main/java/com/github/lukaspili/reactivebilling/observable/GetPurchasesObservable.java

```diff
@@ -33,7 +36,10 @@ protected void onBillingServiceReady(BillingService billingService, Observer<? s
         try {
             observer.onNext(billingService.getPurchases(purchaseType, continuationToken));
             observer.onCompleted();
-        } catch (RemoteException e) {
+        } catch (RemoteException | JSONException e) {
+            if(e instanceof JSONException) {
```

Comment:
Two things:

1. I'm not suggesting getting rid of the log message, I'm suggesting that if there are different actions in the case of different exception types, then having multiple catch blocks is clearer than collapsing them into one so they can be separated by `instanceof` checks.

2. Again, not suggesting removing the logging for `JSONException`s, rather saying log any exception.